src/test/java/com/audiogalaxy/audiogalaxy/UserServiceTest.java

```diff
@@ -83,4 +83,16 @@ public void testDeactivateUserAccountUnsuccessfully() {
 
         Assertions.assertThrows(InformationNotFoundException.class, () -> userService.setUserToInactive());
     }
+
+    @Test
+    @DisplayName("ensure user is the correct user to update username and not blank")
```

Comment:
no need to ensure that the user is the correct user since we're updating the currently logged in user

Reply:
Updated code to include test case for blank and valid username.

Reply:
Pushed the refactored code.  Thanks.